src/system/SystemPacketBuffer.h

```diff
@@ -632,6 +633,44 @@ inline PacketBufferHandle PacketBuffer::Last()
     return PacketBufferHandle::Hold(p);
 }
 
+/**
+ * BufBound backed by packet buffer.
+ */
+class PacketBufBound : public BufBound
+{
+public:
+    /**
+     * Constructs a BufBound that writes into a newly allocated packet buffer.
+     *
+     *  If no memory is available, or if the size requested is too large, then \c IsNull() will be true.
+     *  Otherwise, it is guaranteed that the available space is no less than \a aAvailableSize.
+     *
+     *  @param[in]  aAvailableSize  Minimim number of octets to allocate after the cursor.
+     *  @param[in]  aReservedSize   Minimum number of octets to reserve behind the cursor.
```

Comment:
is this reserve 'before' the cursor?

one argument says 'allocate after' and one says 'reserve behind' ... they ounds the same to me, so I assume reserve is at the front of the buffer. If I am wrong, maybe some more wording will help.

Reply:
Descends from the original Weave `PacketBuffer::New()` comment. I'll try to think of a better wording, and for now, redirect this one to `PacketBufferNew.`